Formula/ansible@2.9.rb

```diff
@@ -0,0 +1,695 @@
+class AnsibleAT29 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automate deployment, configuration, and upgrading"
+  homepage "https://www.ansible.com/"
+  url "https://files.pythonhosted.org/packages/f0/03/4547c6afee926b842a288b12036a7bddce4f7f661b6797414eaee6ac2ccc/ansible-2.9.17.tar.gz"
+  sha256 "d78b46d647d91dd478ba7f43baf36ffa27f9a5992b826b16a6cc15cf993f19dc"
+  license "GPL-3.0-or-later"
+
+  # The newest PyPI release won't be a 2.9.x version, so we match versions from
+  # the links in the "Release History" section.
+  livecheck do
+    url :stable
+    regex(%r{href=.*?/project/ansible/v?(2\.9(?:\.\d+)*)/?["' >]}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/project/ansible/v?(2\.9(?:\.\d+)*)\/?["' >]}i)
```
You missed this from Sam's requested changes.

Reply:
Whoops, this was unnecessary and audit doesn't like it. Fixed.

Reply:
Yep, I meant to unescape that after I converted it from the `/.../` syntax to `%r{...}` but forgot. `/?` is correct.

Reply:
Forgot to explicitly mention that I removed it.
Never mind, brand new formula :)

Reply:
There was no need; I should've realised it. Alas, these `livecheck` `regex`es are voodoo to me. 😅